packages/concerto-vocabulary/lib/vocabulary.js

```diff
@@ -72,6 +65,37 @@ class Vocabulary {
         return this.content.namespace;
     }
 
+    /**
+     * Validates a locale
+     * @param {string} locale the locale to validate
+     * @throws {Error} if the locale is invalid
+     * @private
+     */
+    static validateLocale(locale) {
+        if(!locale) {
+            throw new Error('A vocabulary must specify a locale');
+        }
+        new Intl.Locale(locale);
+        if(locale !== locale.toLowerCase()) {
+            throw new Error('Locale should be lowercase with dashes');
+        }
+    }
+
+    /**
+     * Returns true if the locale is valid
+     * @param {string} locale the locale to validate
+     * @returns {boolean} true if the locale is valid
+     */
+    static isValidLocale(locale) {
```

Comment:
I think both the methods `validateLocale` and `isValidLocale` have similar functionality. Why not return `true` from `validateLocale` itself.

Reply:
The idea was to return true or false instead of throwing exception in case of invalid locale

Reply:
removed this method